packages/image_picker/image_picker_android/pigeons/messages.dart

```diff
@@ -30,6 +30,20 @@ class ImageSelectionOptions {
   int quality;
 }
 
+class MediaSelectionOptions {
+  MediaSelectionOptions({
+    required this.imageSelectionOptions,
+    required this.allowMultiple,
+    this.usePhotoPicker = false,
+  });
+
+  ImageSelectionOptions imageSelectionOptions;
+
+  bool allowMultiple;
+
+  bool usePhotoPicker;
```

Comment:
Optional: Consider extracting a sub-object for these two bools (e.g., GeneralSelectionOptions), which are used in every method.

Reply:
would changing the other methods to use this be a breaking change?

Reply:
With the per-package communication channel structure we use everywhere now, changing a Pigeon API definition is an internal implementation detail, so is not breaking.